examples/cron/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+use worker::*;
+
+#[event(scheduled)]
+async fn scheduled(_evt: ScheduledEvent, _env: Env, _ctx: ScheduleContext) {
+    wasm_rs_dbg::dbg!("Hello cron!");
```

Comment:
Typically we just use the [`console_log`](https://docs.rs/worker/latest/worker/macro.console_log.html) macro to avoid adding an extra dependency.